py/src/call.py

```diff
@@ -402,6 +430,29 @@ def maybe_pending_deployed(deployed_contracts):
     )
 
 
+def maybe_pending_nonces(nonces):
+    if nonces is None:
+        return None
+
+    # accept a map addr => nonce
+    return dict(
+        map(
+            lambda x: (
+                int_param(x[0]),
+                int_param(x[1]),
+            ),
+            nonces.items(),
+        )
+    )
```

Comment:
Maybe something like this is more readable?

```python
dict((int_param(contract_address), int_param(nonce)) for (contract_address, nonce) in nonces.items())
```

Reply:
I can try switch all of these to list comprehensions, but ... these don't really come natural for me.

Reply:
I understand that. I think this improves readability primarily because you're giving things names: instead of `[0]` and `[1]` you now have `contract_address` and `nonce`.

Reply:
In 58b32631